src/drivers/Cst816s.cpp

```diff
@@ -32,6 +32,18 @@ bool Cst816S::Init() {
   twiMaster.Read(twiAddress, 0xa7, &dummy, 1);
   vTaskDelay(5);
 
+  static constexpr uint8_t maxRetries = 3;
+  bool isDeviceOk = false;
+  uint8_t retries = 0;
+  do {
+    isDeviceOk = CheckDeviceIds();
+    retries++;
+  } while(!isDeviceOk && retries < maxRetries);
+
+  if(!isDeviceOk) {
```

Comment:
Hasn't been formatted :)

Reply:
I'm not sure what's wrong here?

Reply:
It would be `if (!isDeviceOk) {` after it has been formatted

Reply:
Oh I forgot we enforced the space after the `if`. Should be ok now :)